src/BizHawk.Client.EmuHawk/tools/Watch/RamWatch.cs

```diff
@@ -494,6 +507,22 @@ private void EditWatch(bool duplicate = false)
 			}
 		}
 
+		private void PokeAddress()

```

Comment:
I think the diff would be smaller if this was beside `PokeAddressMenuItem_Click`.